### PR TITLE
Add web animation polyfill

### PR DESCRIPTION
**Pull request by jonathan-fielding**
Hey

So I have added the web animation polyfill 

Thanks

Jonathan

### CODE REVIEW

File: polyfills/WebAnimations/polyfill.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+// Copyright 2014 Google Inc. All rights reserved.
```
Comment (JakeChampion) [inline polyfills/WebAnimations/polyfill.js]: With automatically imported polyfills, we ignore them in git. Could you add this file to our .gitignore please?

File: polyfills/WebAnimations/config.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+		"safari": "9 - *",
+		"chrome": "* - 38",
+		"ios_saf": "7.1 - *",
+		"firefox_mob": "27 - 50",
+		"android": "<53"
+	},
+	"dependencies": [],
+	"license": "Apache-2.0",
+	"spec": "https://html.spec.whatwg.org/multipage/embedded-content.html#embedded-content",
+	"repo": "https://www.w3.org/TR/web-animations-1/",
```
Comment (JakeChampion) [inline polyfills/WebAnimations/config.json]: The `repo` field is used to specify the repository where the polyfill code can be found, it should be `https://github.com/web-animations/web-animations-js`

File: polyfills/WebAnimations/config.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+		"opera": "11.6 - 42",
+		"safari": "9 - *",
+		"chrome": "* - 38",
+		"ios_saf": "7.1 - *",
+		"firefox_mob": "27 - 50",
+		"android": "<53"
+	},
+	"dependencies": [],
+	"license": "Apache-2.0",
+	"spec": "https://html.spec.whatwg.org/multipage/embedded-content.html#embedded-content",
```
Comment (JakeChampion) [inline polyfills/WebAnimations/config.json]: The spec link for web animations should be `https://w3c.github.io/web-animations/`

File: polyfills/WebAnimations/config.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+		"ie": "10 - *",
+		"ie_mob": "10 - *",
+		"firefox": "27 - 50",
+		"opera": "11.6 - 42",
+		"safari": "9 - *",
+		"chrome": "* - 38",
+		"ios_saf": "7.1 - *",
+		"firefox_mob": "27 - 50",
+		"android": "<53"
+	},
```
Comment (JakeChampion) [inline polyfills/WebAnimations/config.json:12]: browser support section looks good, I looked at https://github.com/web-animations/web-animations-js/blob/dev/docs/support.md#browser-support & https://developer.mozilla.org/en-US/docs/Web/API/Web_Animations_API#Browser_compatibility

**Comment by jonathan-fielding** [comment]
Changes made,

**Review by triblondon — DISMISSED** [review]

**Review by JakeChampion — DISMISSED** [review]

**Comment by JakeChampion** [comment]
@jonathan-fielding It looks to fail in some browsers:
```
 - firefox/44:
    -> WebAnimations
 - ie/14:
    -> WebAnimations
 - ie/13:
    -> WebAnimations
 - ie/11:
    -> WebAnimations
 - ie/10:
    -> WebAnimations
 - safari/9:
    -> WebAnimations
 - android/4.4:
    -> WebAnimations
       http://127.0.0.1:3000/test/tests?mode=targeted&feature=WebAnimations
```

**Comment by JakeChampion** [comment]
@jonathan-fielding It looks to fail in some browsers:
```
 - firefox/44:
    -> WebAnimations
 - ie/14:
    -> WebAnimations
 - ie/13:
    -> WebAnimations
 - ie/11:
    -> WebAnimations
 - ie/10:
    -> WebAnimations
 - safari/9:
    -> WebAnimations
 - android/4.4:
    -> WebAnimations
       http://127.0.0.1:3000/test/tests?mode=targeted&feature=WebAnimations
```

**Comment by jonathan-fielding** [comment]
Have changed the test, turns out the body hasn't loaded when your tests are run so document.body is null. Have moved the test to run on document.head so that test is able to pass

**Comment by jonathan-fielding** [comment]
Also wanted to point this out, not sure who jonburla is?
 
screen shot 2017-04-02 at 20 56 04

Edit: realised the problem, got a new computer but didnt configure Git, updated the commits

**Review by onishiweb — APPROVED** [review]

**Review by JakeChampion — APPROVED** [review]

**Comment by JakeChampion** [comment]
Hi @jonathan-fielding , this is now available for testing on https://qa.polyfill.io/ 👍

**Comment by jonathan-fielding** [comment]
Yay 👍

**Comment by jonathantneal** [comment]
Well done, @jonathan-fielding

**Pull request merged**